job_spec/INSAR_MULTI_BURST.yml

```diff
@@ -0,0 +1,85 @@
+INSAR_MULTI_BURST:
+  required_parameters:
+    - reference
+    - secondary
+  parameters:
+    reference:
+      default:  '""'
+      api_schema:
+        type: array
+        minItems: 1
+        maxItems: 50
+        example:
+          - S1_136231_IW2_20200604T022312_VV_7C85-BURST S1_136232_IW2_20200604T022315_VV_7C85-BURST
+        items:
+          description: Name of the reference Sentinel-1 SLC IW burst granule to process
+          type: string
+          pattern: '^((S1_\d{6}_IW\d_\d{8}T\d{6}_[VH]{2}_([\dA-F]){4}-BURST)($|[ \t]+))+$'
+          minLength: 43
+          example: S1_136231_IW2_20200604T022312_VV_7C85-BURST S1_136232_IW2_20200604T022312_VV_7C85-BURST
```

Comment:
```suggestion
          example:
          - S1_136231_IW2_20200604T022312_VV_7C85-BURST
          - S1_136232_IW2_20200604T022312_VV_7C85-BURST
```

Reply:
Except let's use what's specified in the readme: https://github.com/ASFHyP3/hyp3-isce2/pull/240/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5